sensor/common/installmethod/install.go

```diff
@@ -0,0 +1,34 @@
+package installmethod
+
+import (
+	storage "github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	installMethod string
+	mutex         sync.RWMutex
+)
+
+// Get returns the installation method.
+func Get() string {
+	mutex.RLock()
+	defer mutex.RUnlock()
+
+	return installMethod
+}
+
+// Set sets the installation method based on the managedBy property.
+func Set(value storage.ManagerType) {
+	mutex.Lock()
+	defer mutex.Unlock()
+
+	switch value {
```

Comment:
Make sure we handle the default, unknown case if, for whatever reason, we do not get the `ManagerType` correctly.

Reply:
By default we return an empty string, which leads to the label being dropped by Prometheus.